src/checkbox/_checkbox.scss

```diff
@@ -43,15 +43,10 @@
   .mdl-checkbox.is-upgraded & {
     // Hide input element, while still making it respond to focus.
     position: absolute;
-    width: 0;
-    height: 0;
+    z-index: -1;
     margin: 0;
     padding: 0;
     opacity: 0;
-    -ms-appearance: none;
-    -moz-appearance: none;
-    -webkit-appearance: none;
-    appearance: none;
```

Comment:
As I've experimented - both `width`, `height` equal 0 and `appearance: none` prevents error bubble in chrome, but unfortunately I don't know why :)